src/tespy/components/components.py

```diff
@@ -586,6 +586,12 @@ def initialise_fluids(self, nw):
     def convergence_check(self, nw):
         return
 
+    def exergy_balance(self):
```

Comment:
For consistency, all exergy balance methods should have the same set of parameters (`self` and `Tamb`).